Pinta/Dialogs/DocumentPropertiesDialog.cs

```diff
@@ -0,0 +1,202 @@
+        }
+        
+        private void Build ()
+        {
+            DefaultWidth = 400;
+            DefaultHeight = 300;
+            BorderWidth = 6;
+
+            VBox tBox = new VBox ();// HACK handwriting GUI code sucks
+            tBox.Spacing = 10;
+
+            Notebook notebook = new Notebook ();
+            notebook.BorderWidth = 6;
+            // General Page
+////            notebook.AppendPage (new Button ("1"), new Label ("General"));
+            // Description Page
+            notebook.AppendPage (tBox, new Label (Catalog.GetString ("Description")));
+////            notebook.AppendPage (new Button ("3"), new Label ("Custom"));            
+////            notebook.AppendPage (new Button ("4"), new Label ("Advanced"));
+
+            VBox.Spacing = 10;
+            VBox.PackStart (notebook, true, true, 0);
+
+            // handwriting any GUI code is ugly, GTK especially so
+            Table table = new Table (5, 2, false);
+            tBox.Add (table);
+            uint xPad = 6;
+            uint yPad = 6;
+            
+////    table1.Attach (Widget, leftAttach, rightAttach, topAttach, bottomAttach,
+////    xOptions, yOptions, xPadding, yPadding );
+            Label authorLabel = new Label (Mono.Unix.Catalog.GetString ("_Author:"));
+            table.Attach(authorLabel, 0, 1, 0, 1, AttachOptions.Fill, 0, xPad, yPad);
+            
+            authorEntry = new Entry ();
+            authorLabel.MnemonicWidget = authorEntry;
+            table.Attach(authorEntry, 1, 2, 0, 1, AttachOptions.Fill, 0, xPad, yPad);
+
+            Label titleLabel = new Label (Mono.Unix.Catalog.GetString ("_Title:"));
+            table.Attach(titleLabel, 0, 1, 1, 2, AttachOptions.Fill, 0, xPad, yPad);
+
+            titleEntry = new Entry ();
+            titleLabel.MnemonicWidget = titleEntry;
+            table.Attach(titleEntry, 1, 2, 1, 2, AttachOptions.Fill, 0, xPad, yPad);
+
+            var box3 = new HBox ();
+
+            Label subjectLabel = new Label (Mono.Unix.Catalog.GetString ("_Subject:"));
+            table.Attach(subjectLabel, 0, 1, 2, 3, AttachOptions.Fill, 0, xPad, yPad);
+            
+            subjectEntry = new Entry ();
+            subjectLabel.MnemonicWidget = subjectEntry;
+            table.Attach(subjectEntry, 1, 2, 2, 3, AttachOptions.Fill, 0, xPad, yPad);
+
+            Label keywordsLabel = new Label (Mono.Unix.Catalog.GetString ("_Keywords:"));
+            table.Attach(keywordsLabel, 0, 1, 3, 4, AttachOptions.Fill, 0, xPad, yPad);
+            
+            keywordsEntry = new Entry ();
+            keywordsLabel.MnemonicWidget = keywordsEntry;
+            table.Attach(keywordsEntry, 1, 2, 3, 4, AttachOptions.Fill, 0, xPad, yPad);
+
+            Label commentsLabel = new Label (Mono.Unix.Catalog.GetString ("Co_mments:"));
+            table.Attach(commentsLabel, 0, 1, 4, 5, AttachOptions.Fill, 0, xPad, yPad);
+            
+            // TODO ideally this would be a scrolled box
```

Comment:
I think Pinta's error dialog would provide an example of a scrollable text field.